attestation-agent/attester/src/lib.rs

```diff
@@ -72,12 +79,26 @@ pub enum InitDataResult {
     Unsupported,
 }
 
+pub(crate) type TeeEvidence = serde_json::Value;
+pub(crate) type TeeClass = String;
```

Comment:
Smple type aliases can be confusing since they signal that the type is something more than it actually is. TeeClass is just a String without any verification. If the tee class should be a constrained value, then it can be an (open) enum or a Newtype `struct TeeClass(String)` that will only be produced by some validation. If it's not constrained it should be just a String I think.

Reply:
I am basically just following the style of the TeeParsedClaims alias in Trustee. I don't have strong feelings about this. I don't think simple type alises are necessarily a bad thing. It can make things more descriptive even if it adds an extra trip to `mod.rs` where you say wait it's just a string...

Reply:
but what a string represents should be reflected in the variable name, no? 

a type alias is useful to shorten complex types, but if we want to e.g. signify that a TeeClass cannot be an arbitrary string, but somehow constrained a newtype is a good solution (the newtype would be returned by a fn that asserts the constraint)